### ISSUE TITLE
[Branch 5.10.82] Patch fail to apply since 5.10.117

### ISSUE THREAD

**Issue by Kal42** [sfjro/aufs-standalone#19]
Hi,

Since release 5.10.117, patch from branch 5.10.82 does not apply anymore.

The reject one is : 
```
Applying aufs5-standalone.patch using patch:
patching file fs/file_table.c
Hunk #2 FAILED at 377.
1 out of 2 hunks FAILED -- saving rejects to file fs/file_table.c.rej
```

**Comment by sfjro** [comment]
Hello Kal42,

Kal42:
> Since release 5.10.117, patch from branch 5.10.82 does not apply anymore.
>
> The reject one is : 
> ```
> Applying aufs5-standalone.patch using patch:
> patching file fs/file_table.c
> Hunk #2 FAILED at 377.
> 1 out of 2 hunks FAILED -- saving rejects to file fs/file_table.c.rej
> ```

Thanks for the report.
I've just confirmed that
e68b60ae29de 2022-05-18 SUNRPC: Ensure we flush any closed sockets before xs_xprt_free()
was backported to v5.10.117.
I will make and release aufs5.10.117 branch.

J. R. Okajima

**Comment by sfjro** [comment]
------- Blind-Carbon-Copy

From: "J. R. Okajima" ***@***.***>
To: ***@***.***
Subject: aufs5 GIT release (v5.19-rc5 and aufs5.10.117)
MIME-Version: 1.0
Content-Type: text/plain; charset="us-ascii"
Content-ID: ***@***.***>
Date: Mon, 11 Jul 2022 00:31:12 +0900
Message-ID: ***@***.***>

- - this aufs5.x-rcN branch is for linux-v5.19-rc5, but aufs files are not
  changed since last release.
- - new aufs5.10.117 branch is released for linux-v5.10.117.
  it is a small change to follow new fs/file_table.c,
  repoted by Kal42 on https://github.com/sfjro/aufs5-standalone/issues/19

J. R. Okajima

- ----------------------------------------
- - aufs5-linux.git
  nothing new except aufs5.10.117 branch

- - aufs5-standalone.git
  ditto

- - aufs-util.git
  nothing

------- End of Blind-Carbon-Copy

**Comment by Kal42** [comment]
It's good, thanks.
